daisy_workflows/build-publish/windows/windows-server-next-dc-uefi.publish.json

```diff
@@ -0,0 +1,47 @@
+{{/*
+  Template to publish Windows server UEFI images.
+  By default this template is setup to publish to the 'gce-image-builder'
+  project, the 'environment' variable can be used to publish to 'test', 'prod',
+  or 'staging'.
+  DeleteAfter is set to 180 days for all environments other than prod where no
+  time period is set.
+*/}}
+{
+  "Name": "windows-server-next-dc-uefi",
+  {{$work_project := printf "%q" "gce-image-builder" -}}
+  {{$endpoint := `"https://www.googleapis.com/compute/alpha/projects/"` -}}
+  {{$delete_after := `"24h*30*6"` -}}
+  {{if eq .environment "prod" -}}
+  "WorkProject": {{$work_project}},
+  "PublishProject": "windows-cloud",
+  "ComputeEndpoint": {{$endpoint}},
+  {{- else if eq .environment "internal" -}}
+  "WorkProject": {{$work_project}},
+  "PublishProject": "google.com:windows-internal",
+  "ComputeEndpoint": {{$endpoint}},
+  "DeleteAfter": {{$delete_after}},
+  {{- else -}}
+  "WorkProject": {{$work_project}},
+  "PublishProject": "bct-prod-images",
+  "ComputeEndpoint": {{$endpoint}},
+  "DeleteAfter": {{$delete_after}},
+  {{- end}}
+  {{$guest_features := `["MULTI_IP_SUBNET","UEFI_COMPATIBLE","VIRTIO_SCSI_MULTIQUEUE","GVNIC","WINDOWS"]` -}}
+  {{$time := trimPrefix .publish_version "v"}}
+  "Images": [
+    {
+      "Prefix": "windows-server-next-dc",
+      "Family": "windows-next",
+      "Description": "Microsoft, Windows Server, next Datacenter, Server with Desktop Experience, x64 built on {{$time}}",
+      "Architecture": "X86_64",
+      "Licenses": [
+        {{if eq .environment "internal" -}}
+        "projects/google.com:windows-internal/global/licenses/internal-windows"
+        {{- else -}}
+        "projects/windows-cloud/global/licenses/windows-server-2022-byol"
```

Comment:
Next won't always be Server 2022, right? Will we have to update this with whatever 202x comes, like in 2026?